doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/layouts/ZStackExamples.swift

```diff
@@ -0,0 +1,155 @@
+    }
+}
+
+struct ZStackTopLeadingAlignment: View {
+    var body: some View {
+        NavigationStack {
+            ZStack(alignment: .topLeading) {
+                Rectangle().fill(.red).frame(width: 200, height: 200)
+                Rectangle().fill(.green).frame(width: /*@START_MENU_TOKEN@*/100/*@END_MENU_TOKEN@*/, height: 100)
+                Rectangle().fill(.blue).frame(width: 50, height: 50)
+            }
+        }.navigationTitle("ZStack Top Leading Alignment")
+    }
+}
+
+struct ZStackTopTrailingAlignment: View {
+    var body: some View {
+        NavigationStack {
+            ZStack(alignment: .topTrailing) {
+                Rectangle().fill(.red).frame(width: 200, height: 200)
+                Rectangle().fill(.green).frame(width: /*@START_MENU_TOKEN@*/100/*@END_MENU_TOKEN@*/, height: 100)
+                Rectangle().fill(.blue).frame(width: 50, height: 50)
+            }
+        }.navigationTitle("ZStack Top Trailing Alignment")
+    }
+}
+
+struct ZStackCenterAlignment: View {
+    var body: some View {
+        NavigationStack {
+            ZStack(alignment: .center) {
+                Rectangle().fill(.red).frame(width: 200, height: 200)
+                Rectangle().fill(.green).frame(width: /*@START_MENU_TOKEN@*/100/*@END_MENU_TOKEN@*/, height: 100)
+                Rectangle().fill(.blue).frame(width: 50, height: 50)
+            }
+        }.navigationTitle("ZStack Center Alignment")
+    }
+}
+
+struct ZStackBottomAlignment: View {
+    var body: some View {
+        NavigationStack {
+            ZStack(alignment: .bottom) {
+                Rectangle().fill(.red).frame(width: 200, height: 200)
+                Rectangle().fill(.green).frame(width: /*@START_MENU_TOKEN@*/100/*@END_MENU_TOKEN@*/, height: 100)
+                Rectangle().fill(.blue).frame(width: 50, height: 50)
+            }
+        }.navigationTitle("ZStack Bottom Alignment")
+    }
+}
+
+struct ZStackBottomLeadingAlignment: View {
+    var body: some View {
+        NavigationStack {
+            ZStack(alignment: .bottomLeading) {
+                Rectangle().fill(.red).frame(width: 200, height: 200)
+                Rectangle().fill(.green).frame(width: /*@START_MENU_TOKEN@*/100/*@END_MENU_TOKEN@*/, height: 100)
+                Rectangle().fill(.blue).frame(width: 50, height: 50)
+            }
+        }.navigationTitle("ZStack Bottom Leading Alignment")
+    }
+}
+
+struct ZStackBottomTrailingAlignment: View {
+    var body: some View {
+        NavigationStack {
+            ZStack(alignment: .bottomTrailing) {
+                Rectangle().fill(.red).frame(width: 200, height: 200)
+                Rectangle().fill(.green).frame(width: /*@START_MENU_TOKEN@*/100/*@END_MENU_TOKEN@*/, height: 100)
+                Rectangle().fill(.blue).frame(width: 50, height: 50)
+            }
+        }.navigationTitle("ZStack Bottom Trailing Alignment")
+    }
+}
+
+
+struct ZStackDemo_Previews: PreviewProvider {
```

Comment:
Let's move these previews beneath the view they apply to.

Reply:
Done! Added previews just below the layouts.